Bottom: 85141f44ce229ea9f199d04b77a25949f196b092
Top:    55d13f355f516739dabb7fde56cf6b52eb9fd5f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 21:59:57 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
